specs/Spec.Hash.Definitions.fst

```diff
@@ -37,15 +37,25 @@ type hash_alg =
   | Blake2S
   | Blake2B
   | SHA3_256
+  | SHA3_224
```

Comment:
Could you add a comment along the lines of "For backwards-compatibility with existing C clients when only SHA3_256 was available, we put the SHA3_224 constructor after SHA3_256" to ensure that we do not change the order in future refactorings?

Reply:
Fixed in 2ce8329cd6 which is part of the subsequent PR.